packages/grafana-ui/src/components/Input/Input.mdx

```diff
@@ -21,7 +21,7 @@ To add more context to the input you can add either text or an icon before or af
 
 ## Usage in forms with Field
 
-`Input` should be used with the `Field` component to get labels and descriptions. It should also be used for validation. See the `Field` component for more information.
+`Input` should be used with the `Field` component to get labels and descriptions. It should also be used for validation by using the `required` attribute. See the `Field` component for more information.
```

Comment:
```suggestion
`Input` should be used with the `Field` component to get labels and descriptions. It can also be used for validation by using the `required` attribute. See the `Field` component for more information.
```
Align this with the TextArea perhaps?